src/status_im/contexts/wallet/collectible/tabs/overview/view.cljs

```diff
@@ -0,0 +1,73 @@
+(ns status-im.contexts.wallet.collectible.tabs.overview.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme]
+    [react-native.core :as rn]
+    [status-im.contexts.wallet.collectible.tabs.overview.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- traits-section
+  [traits]
+  (when (pos? (count traits))
+    [rn/view
+     [quo/section-label
+      {:section         (i18n/label :t/traits)
+       :container-style style/traits-title-container}]
+     [rn/flat-list
+      {:render-fn               (fn [{:keys [trait-type value]}]
+                                  [quo/data-item
+                                   {:description     :default
+                                    :card?           true
+                                    :status          :default
+                                    :size            :default
+                                    :title           trait-type
+                                    :subtitle        value
+                                    :container-style style/traits-item}])
+       :data                    traits
+       :key                     :collectibles-list
+       :key-fn                  :id
+       :num-columns             2
+       :content-container-style style/traits-container}]]))
+
+(defn- info
+  [chain-id]
+  (let [network         (rf/sub [:wallet/network-details-by-chain-id
+                                 chain-id])
+        network-keyword (get network :network-name)
+        network-name    (string/capitalize (name network-keyword))]
```

Comment:
The network keyword isn't necessarily the user shown network name, we should have a function that takes network keywords and return network names, e.g. if we transform `:optimism` using `name` and `capitalize` it'll work, but not if we have `:arb1` or if we have a network with dashes in the keyword name.

Also this value should be returned by the subscription, a similar suggestion to what @J-Son89 said for the code below

Reply:
Hey guys, I recently had to fix a potential bug because of a nil value passed to `name`. See my comment and commit here https://github.com/status-im/status-mobile/pull/18276#discussion_r1435225346

The thing is that `name` throws if the value passed to it is nil. See CLJS source code:

```clojure
(defn name
  "Returns the name String of a string, symbol or keyword."
  [x]
  (if (implements? INamed x)
    (-name x)
    (if (string? x)
      x
      (throw (js/Error. (str "Doesn't support name: " x))))))
```

Nil doesn't implement `INamed` and it's not a string, so an exception is thrown.

Therefore, we need to be careful with `name` because it doesn't work well with the idea of *nil punning*.